qdrant-landing/content/documentation/concepts/hybrid-queries.md

```diff
@@ -32,11 +32,25 @@ One of the most common problems when you have different representations of the s
 For example, in text search, it is often useful to combine dense and sparse vectors get the best of semantics,
 plus the best of matching specific words.
 
-There are many ways to fuse the results. One versatile method is <a href=https://plg.uwaterloo.ca/~gvcormac/cormacksigir09-rrf.pdf target="_blank">Reciprocal Rank Fusion (RRF)</a>, 
-which considers the positions of each of points in the results, and boosts the ones that appear closer to the top in multiple queries.
+Qdrant currently has two ways of combining the results from different queries:
 
+- `rrf` - 
+<a href=https://plg.uwaterloo.ca/~gvcormac/cormacksigir09-rrf.pdf target="_blank">
+Reciprocal Rank Fusion
+</a>
 
-Here is an example of RRF for a query containing two prefetches against different named vectors configured to respectively hold sparse and dense vectors. 
+  Considers the positions of results within each query, and boosts the ones that appear closer to the top in multiple of them.
+  
+- `dbsf` - 
+<a href=https://medium.com/plain-simple-software/distribution-based-score-fusion-dbsf-a-new-approach-to-vector-search-ranking-f87c37488b18 target="_blank">
```

Comment:
Do you maybe have a reference that is not on `medium.com`?
In order to make sure it is accessible by everyone anytime.

Reply:
this link is also weaviate affiliated

Reply:
But everybody is using is :man_shrugging: